testing/src/test/java/org/oppia/android/testing/story/StoryProgressTestHelperTest.kt

```diff
@@ -1131,16 +2137,42 @@ class StoryProgressTestHelperTest {
   }
 
   @Test
-  fun testMarkChapterRecentlyPlayed_oneOneProfile_notStartedOnOtherProfile() {
-    storyProgressTestHelper.markRecentlyPlayedTestTopic0Story0Exp0(
+  fun testMarkChapterAsStartedNotCompleted_oneOneProfile_notStartedOnOtherProfile() {
```

Comment:
```suggestion
  fun testMarkChapterAsStartedNotCompleted_onOneProfile_notStartedOnOtherProfile() {
```

Please fix this (preexisting) typo in other similar test names.

Reply:
done